projects/extension/src/background/index.ts

```diff
@@ -33,22 +35,61 @@ declare let window: Background
 window.manager = publicManager
 
 const l = logger("Extension")
+/**
+ * The amount of minutes that the DatabaseContentAlarm
+ * will rerun
+ */
+const periodInMinutes = 5
```

Comment:
There wouldn't be a need for a comment or for a variable definition if we just inlined the value in the configuration Object (which has the same name and it's quite self-descriptive).